docker-entrypoint.sh

```diff
@@ -2,8 +2,9 @@
 set -e
 
 if [[ "$*" == npm*start* ]]; then
-	for dir in "$GHOST_SOURCE/content"/*/; do
-		targetDir="$GHOST_CONTENT/$(basename "$dir")"
+	baseDir="$GHOST_SOURCE/content"
+	for dir in "$baseDir"/*/ "$baseDir"/themes/*/; do
```

Comment:
@adam-beck what I mean in https://github.com/docker-library/ghost/pull/26#issuecomment-190332940 is that this line now explicitly references `$baseDir/themes/*/` as well as just `$baseDir/*/` so that if we're missing any of the default themes, they get added back (thus enabling the "bind mount my in-development theme" workflow to actually be effective)